lang/test/org/partiql/lang/errors/ParserErrorsTest.kt

```diff
@@ -1286,6 +1286,36 @@ class ParserErrorsTest : SqlParserTestBase() {
         )
     }
 
+    @Test
+    fun callTrimSpecificationMismatch() {
+        // for SQL parser, the logic seems to be if the first token is not one of { NONE | BOTH | LEADING | TRAILING}
+        // treating the token as identifier and expect the trim expression to end
```

Comment:
Can you add a `TODO` here saying that this test case will be removed once we remove the legacy `SqlParser`?

Reply:
Addressed in the new commit. I will open up another issue on treating specification mismatch as a parser error.